tests/lib_injection/test_guardrails.py

```diff
@@ -0,0 +1,183 @@
+import os
+from pathlib import Path
+import re
+import subprocess
+
+import pytest
+
+from tests.contrib.integration_registry.test_contrib_versions import _get_integration_supported_versions
+
+
+internal_contrib_dir = Path(os.path.dirname(__file__)) / ".." / ".." / "ddtrace" / "contrib" / "internal"
+
+anthropic_spec = _get_integration_supported_versions(internal_contrib_dir, "anthropic")["anthropic"]
+pymemcache_spec = _get_integration_supported_versions(internal_contrib_dir, "pymemcache")["pymemcache"]
+pymongo_spec = _get_integration_supported_versions(internal_contrib_dir, "pymongo")["pymongo"]
```

Comment:
any reason for testing these specific integrations?

Reply:
theyre pretty lightweight to install, they have version ranges that we don't support but are also compatible from python 3.8 to 3.13 for those ranges, and the package pypi name equals the module import name, which is nice when formatting the name into the application script.